package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "minim-api-description",
+  "version": "0.1.0",
+  "description": "Minim API Description Namespace",
+  "main": "./lib/api-description.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/refractproject/minim-api-description.git"
+  },
+  "scripts": {
+    "compile": "eslint src test && babel src --optional runtime --out-dir lib",
+    "coverage": "babel-node node_modules/.bin/isparta cover _mocha -- -R spec --recursive",
+    "precoveralls": "npm run coverage",
+    "coveralls": "coveralls <coverage/lcov.info",
```

Comment:
By the way, we should move to code climate (which is way better) sometime later and make sure all of our code is getting checked for test coverage.